codegen/codegen_outofplacebatching.py

```diff
@@ -155,7 +155,7 @@ def parse_return(return_t):
     return tuple([x.strip() for x in m.group(1).split(',')])
 
 def parse_args(args_t):
-    args = args_t.split(',')
+    args = args_t.split(', ')
```

Comment:
There is an assumption made that args are separated with comma-space. Such that I do not reinvent how to parse args in general case with any kind of arg types.

Reply:
This is fine, can you add a comment here regarding the assumption?